cisco_os_shim.gemspec

```diff
@@ -3,31 +3,22 @@ lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'cisco_os_shim/core/version'
 
-plugin_files = Dir['cisco_os_shim-*.gemspec'].map do |gemspec|
-  eval(File.read(gemspec)).files # rubocop:disable Lint/Eval
-end
-plugin_files = plugin_files.flatten.uniq
-
 Gem::Specification.new do |spec|
   spec.name          = 'cisco_os_shim'
   spec.version       = Cisco::Shim::VERSION
-  spec.authors       = ['Glenn Matthews']
+  spec.authors       = ['Rob Gries', 'Alex Hunsberger', 'Glenn Matthews',
+                        'Chris Van Heuveln', 'Mike Wiebe', 'Jie Yang']
```

Comment:
You weren't in the authors list previously for cisco_nxapi - this is the original authors list plus Rob since he fixed Travis. Have you contributed any code to this gem? If so I'm happy to add you.